web/src/pages/people/index.tsx

```diff
@@ -26,12 +28,14 @@ export default function PeoplePage() {
         description={
           isAdmin
             ? "View and manage portal users, including adding via invitation or upload"
-            : isIAO
-              ? "View users in your projects or invite a collaborator"
-              : "You do not have permission to view this page"
+            : isTreOpsStaff
+              ? "View approved researchers"
+              : isIAO
+                ? "View users in your projects or invite a collaborator"
+                : "You do not have permission to view this page"
         }
       />
-      {isAdmin ? <AdminView /> : isIAO && <IAOView />}
+      {isAdmin ? <AdminView /> : isTreOpsStaff ? <TreOpsStaffView /> : isIAO && <IAOView />}
```

Comment:
It looks like these `View` files share some basic common functionality. I wonder if it would be more efficient to fetch the people in `pages/people/index.tsx` and then pass them to all subsequent components. Then you could potentially get rid of all the separate `View` files and have one file that handles the UI based on the different roles.

The danger would be that we start making all these different people `View` files and end up with a lot of files and repeated logic.

Reply:
I also thought about that and wondered if it might be problematic for someone to be able to technically have access to all the users (in the network response) but they only see a subset in the UI. I opted to air on the side of caution but happy to have a conversation about it. It may also be that at _this_ stage they're fairly similar but will grow more different over time

Reply:
there's also the new UserDataTable which is the bulk of the similarity, and then it's really only the fetching that's split out for each view

Reply:
I'd probably just do it in this PR since the work is directly relevant to what the PR is about. Unless you felt strongly about it?